### ISSUE TITLE
Documentation error: GET/ {db} request always has instance_start_time of "0"

### ISSUE THREAD

**Issue by sundaramoorthyjeeva** [apache/couchdb#1422]
## Expected Behavior
I am trying to get the database created date time. Couch DB document guide me to use GET request to get DB information.
http://docs.couchdb.org/en/2.0.0/api/database/common.html

## Current Behavior
I requested the GET with database and appropriate request inputs. I got the response and it has instance_start_time key. The expected thing is I need database created time in milliseconds and this value will be available against instance_start_time key. But this key always gives the value "0"

## Steps to Reproduce (for bugs)
1. http://127.0.0.1:5984/database_name
2. Make request based on the steps mentioned in document
3. Request from Post Man extension and check response.

## Your Environment
* Version used: Couch DB 2.1
* Browser Name and version: Google Chrome and Version 67.
* Operating System and version (desktop or mobile): Windows 7 Desktop

**Comment by wohali** [comment]
This is a documentation error, and relates to clustering.

TL;DR: We don't recommend you rely on this value for anything in CouchDB 2.x or later versions. If you need to store when a database is created, create a document inside the database that includes that information at the time you create a database yourself.

The reason this information is not available on the clustered interface is that, for a default CouchDB cluster setup of 3 nodes with q=8, there are 24 database shards for any given database, each of which has its own instance_start_time.

There is currently no way to get each shard's instance start time through the clustered interface. However, on each node via localhost through the node-local (port 5986), you can get the instance_start_time per shard:

```
 $ curl http://localhost:5986/shards%2F00000000-1fffffff%2Fabc.1530133574
{"db_name":"shards/00000000-1fffffff/abc.1530133574","engine":"couch_bt_engine","doc_count":127,"doc_del_count":0,"update_seq":127,"purge_seq":0,"compact_running":false,"sizes":{"active":31925,"external":3544,"file":90308},"disk_size":90308,"data_size":31925,"other":{"data_size":3544},"instance_start_time":"1530692235563311","disk_format_version":6,"committed_update_seq":127,"compacted_seq":127,"uuid":"953768671309b22f6bf8a5eae62f94a4"}
```

Note again that this value will be different across multiple shards in the same database, as well as across multiple nodes in the same cluster. Only for a q=1 database in a single standalone server will you have a single instance_start_time value, which can and will change after database compaction (which occurs automatically by default.)

**DO NOT USE PORT 5986 FOR REGULAR DATABASE OPERATIONS**. This interface is deprecated in 2.x and will be removed entirely in 3.x. By that time there will be replacements for all potential needs on the clustered interface.

**Comment by nickva** [comment]
The advice for @wohali is correct. We also document since 2.0 that instance_start_time always returns "0"

**Comment by xrobau** [comment]
## Minor update, 2 years later

For those that are chasing issues with this no longer returning 0 in 3.3.0 and higher, and as this issue is high in the google search results, this changed from 'always return zero' to returning the correct utime [in 3.3.0](https://docs.couchdb.org/en/stable/whatsnew/3.3.html), specifically in PR #3901 - however, as far as I'm aware the advice above is still correct - this is only used for replication and should not be consumed by anything external.
